Quick/Configuration/QuickConfiguration.m

```diff
@@ -1,7 +1,10 @@
 #import "QuickConfiguration.h"
+#import "World.h"
 #import <Quick/Quick-Swift.h>
 #import <objc/runtime.h>
 
+QuickWorldBridge
```

Comment:
Placing the forward declarations in header files causes compilation errors--they only compile in implementation files. The macro allows for that. I might be doing something wrong here... :fearful: